docs/architecture/ADR-005-Map-Interaction-System-Consolidation.md

```
@@ -0,0 +1,647 @@
+
+  /** @deprecated Use ContentProvider pattern */
+  export function setCurrentTooltip(content: React.ReactNode) {
+    console.warn('currentTooltipAtom deprecated - use ContentProvider');
+    // Delegate to unified service
+  }
+}
+```
+
+### Migration Checklist
+
+**Phase 1: Infrastructure (Week 1-2)**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)